app/internal/email_use.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
Prefer easier name: Change the filename to "email", and the name of the first function to "send".